kong/db/declarative/init.lua

```diff
@@ -404,23 +405,33 @@ local function export_from_db(emitter, skip_ws)
         table.insert(fks, field_name)
       end
     end
-
+    
     for row, err in kong.db[name]:each(nil, { nulls = true, workspace = null }) do
       if not row then
         kong.log.err(err)
         return nil, err
       end
 
-      for _, foreign_name in ipairs(fks) do
-        if type(row[foreign_name]) == "table" then
-          local id = row[foreign_name].id
-          if id ~= nil then
-            row[foreign_name] = id
+      -- do not export disabled services and associated enitties when skip_disabled_services
+      if skip_disabled_services and name == "services" and not row.enabled then
```

Comment:
```suggestion
      if skip_disabled_entities and (name == "services" or name == "plugins") and not row.enabled then
```

Just a suggestion, but I think you get the idea.

I know this is feature creep, but while we are at it, it probably makes sense to deal with plugins too. Plugins are not referenced by any other table, so plugins don't need same treatment as services that related entities should be skipped too.

Reply:
done